dev_scripts/reproduce.py

```diff
@@ -0,0 +1,166 @@
+#!/usr/bin/env python3
+
+import argparse
+import hashlib
+import logging
+import pathlib
+import stat
+import subprocess
+import sys
+import urllib.request
+
+logger = logging.getLogger(__name__)
+
+DIFFOCI_URL = "https://github.com/reproducible-containers/diffoci/releases/download/v0.1.5/diffoci-v0.1.5.linux-amd64"
```

Comment:
It seems to me that `reproduce.py` requires `diffoci` but only checks out the `amd64` version. Should we also find a way to download it on `arm64` (e.g. on silicon macs)?

Reply:
Yep, we can go with this for now and refine later!